example/http/client/body/Jamfile

```diff
@@ -11,4 +11,5 @@ exe json_client : json_client.cpp
     :
     <variant>coverage:<build>no
     <variant>ubasan:<build>no
+    <library>/boost/json//boost_json
```

Comment:
This causes a link error because we aslo include `<boost/json/src.hpp>` in `json_client.cpp`.

```
compile-c-c++ ..\..\bin.v2\libs\beast\example\http\client\body\msvc-14.1\release\x86_64\cxxstd-17-iso\threading-multi\json_client.obj
json_client.cpp
..\..\boost/json/impl/array.ipp(31): warning C4273: 'empty_': inconsistent dll linkage
..\..\boost/json/array.hpp(96): note: see previous definition of 'private: static boost::json::array::table boost::json::array::empty_'
..\..\boost/json/impl/array.ipp(31): error C2491: 'boost::json::array::empty_': definition of dllimport static data member not allowed
..\..\boost/json/impl/array.ipp(40): warning C4273: 'boost::json::array::table::allocate': inconsistent dll linkage
..\..\boost/json/impl/array.hpp(42): note: see previous definition of 'allocate'
..\..\boost/json/impl/array.ipp(63): warning C4273: 'boost::json::array::table::deallocate': inconsistent dll linkage
...
```